boilerplate/README.md

```diff
@@ -64,72 +66,66 @@ ignite-project
 └── package.json
 ```
 
-The directory structure uses a ["feature first, function second"](https://alligator.io/react/index-js-public-interfaces/) approach to organization. Files are grouped by the feature they are supporting rather than the type of file.
-
-For example: A custom `Button` component would have the main component file, and test, and any assets or helper files all grouped together in one folder.
-
-This is a departure from the previous boilerplate, which grouped files by type (components together, styles together, tests together, images together, etc.). One feature of this new approach is the use of index files which export specific functions from files in the directory to create a public interface for each "thing", or "feature. You'll see that pattern quite a bit in this boilerplate.
-
-
-## ./src directory
+### ./app directory
 
 Included in an Ignite boilerplate project is the src directory. This is a directory you would normally have to create when using vanilla React Native.
 
 The inside of the src directory looks similar to the following:
 
 ```
-src
-├── app
+app
+│── components
 │── i18n
-├── lib
 ├── models
 ├── navigation
+├── screens
 ├── services
 ├── theme
-├── views
+├── utils
+├── app.tsx
+├── environment-variables.tsx
```

Comment:
Do we want to go with `env.tsx` or `environment-variables.tsx`?

Also, should this be `.ts`, not `.tsx`?

Reply:
(I don't mind either way, btw)

Reply:
I decided to leave it as `environment-variables`, since we also have the `environment` model that's part of the MST workflow, so I feel like this will avoid some confusion. I think you're right that it should be `.ts` though.